diff --git a/dist/cjs/index.js b/dist/cjs/index.js
index fd1b6333f5f40a7685400891e8f5f02f59ca8cd6..3f17e8abcbe7dc35cc50f1bbe81b2cedc90d623a 100644
--- a/dist/cjs/index.js
+++ b/dist/cjs/index.js
@@ -296,19 +296,7 @@ function containsEnumOptions(compilerOptions) {
  * This needs to be set explicitly for `resolveModuleName` to select the correct resolution method
  */
 function setModuleResolutionKind(parsedConfig) {
-    const moduleKind = parsedConfig.options.module;
-    const moduleResolution = moduleKind === ModuleKind$1.Node16
-        ? ModuleResolutionKind.Node16
-        : moduleKind === ModuleKind$1.NodeNext
-            ? ModuleResolutionKind.NodeNext
-            : ModuleResolutionKind.NodeJs;
-    return {
-        ...parsedConfig,
-        options: {
-            ...parsedConfig.options,
-            moduleResolution
-        }
-    };
+    return parsedConfig;
 }
 const configCache = new Map();
 /**
diff --git a/dist/es/index.js b/dist/es/index.js
index b40ae0e8326ac49a8e98fbcf66ab85e8acc8a150..fc11a4fcf3cbd973683b7051ee5c6717e2b3f96c 100644
--- a/dist/es/index.js
+++ b/dist/es/index.js
@@ -274,19 +274,7 @@ function containsEnumOptions(compilerOptions) {
  * This needs to be set explicitly for `resolveModuleName` to select the correct resolution method
  */
 function setModuleResolutionKind(parsedConfig) {
-    const moduleKind = parsedConfig.options.module;
-    const moduleResolution = moduleKind === ModuleKind$1.Node16
-        ? ModuleResolutionKind.Node16
-        : moduleKind === ModuleKind$1.NodeNext
-            ? ModuleResolutionKind.NodeNext
-            : ModuleResolutionKind.NodeJs;
-    return {
-        ...parsedConfig,
-        options: {
-            ...parsedConfig.options,
-            moduleResolution
-        }
-    };
+    return parsedConfig;
 }
 const configCache = new Map();
 /**